resources/language/resource.language.de_de/strings.po

```diff
@@ -267,3 +271,27 @@ msgstr "Profil auswählen"
 msgctxt "#30080"
 msgid "There are too many active streams. Please try again later."
 msgstr "Es sind zu viele Streams aktiv. Bitte probiere es später noch einmal."
+
+msgctxt "#30089"
+msgid "UpNext fixed or unavailable end detection duration"
+msgstr "Vorlaufzeit für UpNext wenn \"Statisch\" oder keine Credits/Vorschau"
+
+msgctxt "#30090"
+msgid "UpNext integration"
+msgstr "UpNext Integration"
+
+msgctxt "#30091"
+msgid "At credits start, if nothing after"
+msgstr "Zu Beginn der Credits, wenn danach keine Vorschau kommt"
```

Comment:
Probably need to refine this, I wasn't sure what the english text was trying to tell me. :D

Reply:
As I wrote in a previous comment, I change it to something more understandable.

> In fact, it may be named like that (because, this is what it does):
> 
> "Show UpNext dialog at":
> - "credits start"
> - "preview start"
> - "fixed time before the end"
> -  "never (disabled)"